test/active_record_test.rb

```diff
@@ -329,7 +329,7 @@ class Measured::Rails::ActiveRecordTest < ActiveSupport::TestCase
   end
 
   test "assigning a number with more significant digits than permitted by the column precision does not raise exception when it can be rounded to have lesser significant digits per column's scale" do
-    assert_nothing_raised Measured::Rails::Error do
+    assert_nothing_raised do
```

Comment:
Can we remove `assert_nothing_raised` now, and let unexpected exceptions bubble up to the test runner?